.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 os: linux
-jdk: oraclejdk8
+jdk: openjdk8
```

Comment:
Should this change be included here or would it be better done under a different JIRA ticket? Is this related to OPENNLP-1273?

Reply:
In retrospect, I think it could get its own JIRA ticket. It is not related to OPENNLP-1273, this was an issue with Travis CI failing the machine setup (it could not install Oracle JDK8, though the docs claim the opposite).

Reply:
Lets move it into another PR. And then can merge this for the next release.

Reply:
Done.